diff -Naur --exclude '*~' iv-2.5.1/iv/platforms.ini iv-2.5.1-fix-lib64-build/iv/platforms.ini
--- iv-2.5.1/iv/platforms.ini	2009-02-10 09:00:00.000000000 +0100
+++ iv-2.5.1-fix-lib64-build/iv/platforms.ini	2009-03-14 01:19:46.000000000 +0100
@@ -553,6 +553,210 @@
 			MustExist = Yes
 			DependPath = endeavour2-base-config
 
+# Linux
+#
+Platform = Linux64
+	Description = For most Linux systems
+	PREFIX = /usr/
+	CFLAGS = -Wall -O6 -funroll-loops -fomit-frame-pointer -ffast-math \
+-finline-functions -DNEED_STRCASESTR
+	LIBS = -lm
+	LIB_DIR =
+	CC = cc
+	CPP = c++
+	PlatformSearchPathInclude = /usr/include/
+	PlatformSearchPathInclude = /usr/X11R6/include/
+	PlatformSearchPathInclude = /usr/local/include/
+	PlatformSearchPathInclude = /opt/gnome/include/
+	PlatformSearchPathLib = /lib/64
+	PlatformSearchPathLib = /usr/lib64
+	PlatformSearchPathLib = /usr/X11R6/lib/
+	PlatformSearchPathLib = /usr/local/lib/
+	PlatformSearchPathLib = /opt/gnome/lib/
+	PlatformSearchPathEtc = /etc/
+	PlatformSearchPathEtc = /usr/etc/
+	PlatformSearchPathEtc = /usr/local/etc/
+	PlatformSearchPathBin = /bin/
+	PlatformSearchPathBin = /usr/bin/
+	PlatformSearchPathBin = /usr/X11R6/bin/
+	PlatformSearchPathBin = /usr/local/bin/
+	PlatformSearchPathBin = /usr/games/
+	PlatformSearchPathBin = /opt/gnome/bin/
+	PlatformSearchPathData = /usr/share/icons/
+	PlatformSearchPathData = /usr/share/games/
+	PlatformFeature = debug
+		Description = Just adds -g to the CFLAGS for debugging
+		MustExist = No
+		FeatureCFLAGS = -g
+	PlatformFeature = arch-i486
+		Description = Adds -march=i486 for 486 specific optimizations
+		MustExist = Preferred  
+		FeatureCFLAGS = -march=i486
+		FeatureDepend = i486
+			DependType = Machine
+			MustExist = Yes
+			DependMachine = i486
+	PlatformFeature = arch-i586
+		Description = Adds -march=i586 for Pentium specific optimizations
+		MustExist = Preferred
+		FeatureCFLAGS = -march=i586
+		FeatureDepend = i586
+			DependType = Machine
+			MustExist = Yes
+			DependMachine = i586
+	PlatformFeature = arch-i686
+		Description = Adds -march=i686 for Pentium Pro specific optimizations
+		MustExist = Preferred
+		FeatureCFLAGS = -march=i686
+		FeatureDepend = i686
+			DependType = Machine
+			MustExist = Yes
+			DependMachine = i686
+	PlatformFeature = arch-pentiumpro
+		Description = Adds -march=pentiumpro for Pentium Pro specific optimizations
+		MustExist = Preferred
+		FeatureCFLAGS = -march=pentiumpro
+		FeatureDepend = pentiumpro
+			DependType = Machine
+			MustExist = Yes
+			DependMachine = pentiumpro
+	PlatformFeature = title_image
+		Description = Compiles in the title image (increases\
+ size of binary)
+		MustExist = Preffered
+		FeatureCFLAGS = -DINCLUDE_IV_TITLE_IMAGE
+	PlatformFeature = libgif
+		Description = Uncompressed GIF Library
+		MustExist = Preffered
+		FeatureCFLAGS = -DHAVE_LIBGIF
+		FeatureLIBS = -lgif
+		FeatureDepend = libgif-devel
+			DependType = Header
+			MustExist = Yes
+			DependPath = gif_lib.h
+			DependGrepString = DGifOpenFileName
+	PlatformFeature = libjpeg
+		Description = JPEG Library
+		MustExist = Preffered
+		FeatureCFLAGS = -DHAVE_LIBJPEG
+		FeatureLIBS = -ljpeg
+		FeatureDepend = libjpeg-devel
+			DependType = Header
+			MustExist = Yes
+			DependPath = jpeglib.h
+			DependGrepString = jpeg_create_decompress
+	PlatformFeature = libpng
+		Description = PNG Library
+		MustExist = Preffered
+		FeatureCFLAGS = -DHAVE_LIBPNG `libpng-config --cflags`
+		FeatureLIBS = `libpng-config --libs`
+		FeatureDepend = libpng-devel
+			DependType = Header
+			MustExist = Yes
+			DependPath = png.h
+			DependGrepString = png_create_read_struct
+	PlatformFeature = libmng
+		Description = MNG Library
+		MustExist = Preffered
+		FeatureCFLAGS = -DHAVE_LIBMNG
+		FeatureLIBS = -lmng -lz
+		FeatureDepend = zlib-devel
+			DependType = Header
+			MustExist = Yes
+			DependPath = zlib.h
+			DependGrepString = deflateInit
+		FeatureDepend = libmng-devel
+			DependType = Header
+			MustExist = Yes
+			DependPath = libmng.h
+			DependGrepString = mng_initialize
+	PlatformFeature = libxpm
+		Description = XPM Library
+		MustExist = Preffered
+		FeatureCFLAGS = -DHAVE_LIBXPM
+		FeatureLIBS = -lXpm
+		FeatureLIB_DIR = -L/usr/X11R6/lib/
+		FeatureDepend = libxpm-devel
+			DependType = Header
+			MustExist = Yes
+			DependPath = X11/xpm.h
+			DependGrepString = XpmReadFileToXpmImage
+	PlatformFeature = Imlib
+		Description = Image Library
+		MustExist = Preffered
+		FeatureCFLAGS = `imlib-config --cflags` -DHAVE_IMLIB
+		FeatureLIBS = `imlib-config --libs`
+		FeatureDepend = Imlib-devel
+			DependType = Program
+			MustExist = Yes
+			DependPath = imlib-config
+	PlatformFeature = Imlib-transpixel-fix
+		Description = Bugfix for Imlib transparent pixel loading
+		MustExist = Preffered
+		FeatureCFLAGS = -DUSE_IMLIB_TRANSPIXEL_FIX
+	PlatformFeature = XF86VidMode
+		Description = XF86 Video Mode extension
+		MustExist = Preferred
+		FeatureCFLAGS = -DHAVE_XF86_VIDMODE
+		FeatureLIBS = -lXxf86vm
+		FeatureLIB_DIR = -L/usr/X11R6/lib/
+		FeatureDepend = xf86vidmode-lib
+			DependType = Library
+			MustExist = Yes
+			DependPath = libXxf86vm.a
+			DependGrepString = XF86VidModeQueryExtension
+		FeatureDepend = xf86vidmode-devel
+			DependType = Header
+			MustExist = Yes
+			DependPath = X11/extensions/xf86vmode.h
+			DependGrepString = XF86VidModeQueryExtension
+	PlatformFeature = libXp
+		Description = X Print printing (instead of using lpr for printing)
+		MustExist = Preffered
+		FeatureCFLAGS = -DHAVE_XPRINT
+		FeatureLIBS = -lXp -lXprintUtil
+		FeatureLIB_DIR = -L/usr/X11R6/lib/
+		FeatureDepend = libXp-lib
+			DependType = Library
+			MustExist = Yes
+			DependPath = libXp.so
+			DependGrepString = XpQueryExtension
+		FeatureDepend = libXp-devel
+			DependType = Header
+			MustExist = Yes
+			DependPath = X11/extensions/Print.h
+			DependGrepString = XpQueryExtension
+		FeatureDepend = libXprintUtil-lib
+			DependType = Library
+			MustExist = Yes
+			DependPath = libXprintUtil.so
+			DependGrepString = XpuCheckExtension
+		FeatureDepend = libXprintUtil-devel
+			DependType = Header
+			MustExist = Yes
+			DependPath = X11/XprintUtil/xprintutil.h
+			DependGrepString = XpuCheckExtension
+	PlatformFeature = libgtk
+		Description = GTK+ 1.2.x
+		MustExist = Yes
+		FeatureCFLAGS = `gtk-config --cflags`
+		FeatureLIBS = `gtk-config --libs`
+		FeatureLIB_DIR = `gtk-config --libs`
+		FeatureDepend = gtk-devel
+			DependType = Program
+			MustExist = Yes
+			DependPath = gtk-config
+	PlatformFeature = libendeavour2-base
+		Description = Endeavour Mark II Base Library
+		MustExist = Preffered
+		FeatureCFLAGS = -DHAVE_LIBENDEAVOUR2 `endeavour2-base-config --cflags`
+		FeatureLIBS = `endeavour2-base-config --libs`
+		FeatureLIB_DIR = `endeavour2-base-config --libs`
+		FeatureDepend = endeavour2-base-devel
+			DependType = Program
+			MustExist = Yes
+			DependPath = endeavour2-base-config
+
 
 # Linux - Big Endian
 #
